src/concrete/ml/onnx/ops_impl.py

```diff
@@ -1060,11 +1101,17 @@ def numpy_less_or_equal(
     Args:
         x (numpy.ndarray): Input tensor
         y (numpy.ndarray): Input tensor
+        lsbs_to_remove (Optional[int]]): The `AutoRounder` which determines
+            the number of the least significant bits to remove
 
     Returns:
         Tuple[numpy.ndarray]: Output tensor
     """
 
+    if lsbs_to_remove is not None and lsbs_to_remove > 0:
+        return rounded_comparison(y, x, lsbs_to_remove, operation=lambda x: x >= 0)
```

Comment:
same as above, keep it coherent : swap back x and y, and then put `x <= 0`

Reply:
it's not equivalent.
You can check this notebook, and you can change x - y <= 0, you will see that the test fails.
https://github.com/zama-ai/fhenets_experiments/blob/main/tree_rounding_experiments/impact_of_rounding_on_tree_based_models.ipynb

Reply:
ok looking at the code I think I get it, it's because of this -half that we add to the (x-y)

in that case, please explain it in both a comment here (+ above similar likes for less and for equal) + the `rounded_comparison` docstring. Maybe we'll be able to remove them once we have the truncate feature, but until now it is pretty confusing and will be hard to debug in the future without such information !